website/content/api-docs/peering.mdx

```diff
@@ -42,12 +42,10 @@ The table below shows this endpoint's support for
 - `Partition` `(string: "")` - <EnterpriseAlert inline /> The admin partition that the
   peering token is generated from. Uses `default` when not specified.
 
-- `Datacenter` `(string: "")` - Specifies the datacenter where the peering token is generated. Defaults to the
-  agent's datacenter when not specified.
-
-- `Token` `(string: "")` - Specifies the ACL token to use in the request. Takes precedence
-  over the token specified in the `token` query parameter, `X-Consul-Token` request header,
-  and `CONSUL_HTTP_TOKEN` environment variable.
+- `ServerExternalAddresses` `([]string: <optional>)` - A list of addresses to put
+  into the generated token. This could be used to specify load balancer(s) or
+  external IPs to reach the servers from the dialing side, and will override any server
+  addresses obtained from the "consul" service.
```

Comment:
I think it could be more clear what external IPs or load balancers we're talking about, and I'm not sure anymore that the last sentence adds value to why someone would need this.
```suggestion
into the generated token. You can specify one or more load balancers or external IPs that route external traffic to the cluster's Consul servers. 
```

Reply:
OK, as long as it's clear to users which cluster "this" refers to then I think this is fine.